go-controller/pkg/ovn/master.go

```diff
@@ -765,20 +769,50 @@ func (oc *DefaultNetworkController) addUpdateLocalNodeEvent(node *kapi.Node, nSy
 		}
 	}
 
+	if nSyncs.syncZoneIc && config.OVNKubernetesFeature.EnableInterconnect {
+		if err := oc.zoneChassisHandler.AddLocalZoneNode(node); err != nil {
+			errs = append(errs, err)
+			oc.syncZoneIcFailed.Store(node.Name, true)
+		} else {
+			if err := oc.zoneICHandler.AddLocalZoneNode(node); err != nil {
```

Comment:
haha the func names are same handlers are different, I do see some nice docs over in `pkg/ovn/interconnect` IIUC zoneICHandler does all the NBDB config stuff for the local zone including the remote zone connection pieces and zoneChassisHandler does all the SBDB config stuff for remote/local chassis...

nit: add some code comment here for the logic explanation of this if else with IC and Chassis handlers...